adhocteam/Head-Start-TTADP
# [TTAHUB-216] Regional Dashboard: Turn off specific lines on TTA Hours graph

## Description of change
Added checkboxes and a button control to toggle the lines drawn on the TTA Hours graph.

## How to test
Visit the Regional Dashboard page at /regional-dashboard

Note that the Total TTA Hours graph now has checkboxes and an apply filters button.

Select and/or deselect specific lines and click the button. The traces drawn on the graph should change to reflect your selections.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-216

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
